app/helpers/model_name_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ModelNameHelper
+  def model_name(record)
+    record.class.model_name.name
+  end
+end
```

Comment:
Maybe i can add this method into ApplicationHelper?

Reply:
Looks like nope, because whatever need to `include ApplicationHelper` in some places where i need to use `model_name` method